public/locales/en.json

```diff
@@ -37,6 +45,8 @@
     "visibleInSearch": "Visible In Search",
     "displayImage": "Display Image",
     "enterName": "Enter Name",
+    "toastSuccess": "Congratulation the Organization is created",
+    "toastError": "Talawa-API service is unavailable. Is it running? Check your network connectivity too.",
```

Comment:
Why not use a more appropriate name that represent the message. I see toastError as too ambiguous and could lead to potential confusion for future devs.

Reply:
For the message "Talawa-API service is unavailable. Is it running? Check your network connectivity too." The property name `talawaApiUnavailable` should suffice.

Reply:
Congratulation_the_Organization_is_created could be `createOrgSuccess`